source/Tooltip.cpp

```diff
@@ -151,13 +147,14 @@ Tooltip::Tooltip(int width, Alignment alignment, Direction direction, Corner cor
 	// 10 pixels of padding will be left on either side of the tooltip box.
 	text.SetWrapWidth(width - 20);
 	text.SetAlignment(alignment);
+	UpdateActivationCount();
```

Comment:
If you open a panel that has a tooltip, then from there go to the preferences panel to change the hover time, then pop back to the original panel, it won't receive this update since every panel aside from the preferences panel only sets its hover time on initialization.

I don't think you can actually get this to happen anywhere in the game, though, since you need to go to the menu in order to get to the preferences panel, and the menu can only be accessed by hitting escape from the planet, main, or conversation panels, none of which have tooltips. (Hitting escape in other panels just closes them.)

Reply:
Well it wouldn't be a game restart, it would just be closing and re-opening the panel, since that re-initializes the tooltip.

Reply:
Yeah I know, but if we want the player to close and reopen every panel, a game restart would be the simplest way. The internal details of the Panel architecture aren't directly exposed to the player, so saying "close every panel to make sure it gets updated" wouldn't be clear.

Reply:
Or maybe we could go through the panel stack and have some method on Panel that updates the Tooltip for panels that have them.

Reply:
That would probably be pretty straightforward.